dll/win32/browseui/internettoolbar.cpp

```diff
@@ -718,7 +722,15 @@ HRESULT CInternetToolbar::LockUnlockToolbars(bool locked)
 
     if (locked != fLocked)
     {
+        DWORD dwLocked = locked ? 1 : 0;
+        SHRegSetUSValueW(L"Software\\Microsoft\\Internet Explorer\\Toolbar",
+                         L"Locked",
+                         REG_DWORD,
+                         &dwLocked,
+                         sizeof(DWORD),
```

Comment:
Wouldn't it just work with BOOL, i.e. without additional DWORD variable?

```suggestion
        SHRegSetUSValueW(L"Software\\Microsoft\\Internet Explorer\\Toolbar",
                         L"Locked",
                         REG_DWORD,
                         &locked,
                         sizeof(locked),
```

Reply:
In my opinion it is better with it being DWORD.